dashboard/about.md

```diff
@@ -64,27 +64,19 @@ Data for the dashboard is pulled from these sources on Mondays and Tuesdays.
 *   **Epidemiological week (MMWR week)**
     
     Week that starts on a Sunday. If the day on which the forecast is being made is a Sunday or Monday, the next epidemiological week is the week that starts on that Sunday (going back a day if it is Monday). If the forecast is being made on Tuesday-Saturday, the next epidemiological week is the week that starts on the subsequent Sunday, following [CDC convention](https://wwwn.cdc.gov/nndss/document/MMWR_week_overview.pdf).
-    
-*   **Point Forecast**
-    
-  The value that each forecaster picks as their “most likely” prediction. Usually this is the median (50% quantile prediction), but forecasters can specify alternative Point Forecasts different from the median. 
-  
-*   **Geo Type**
-    
-    States, territories or U.S. as a nation
 
 #### **Dashboard Inclusion Criteria**
+A forecast is only included if all the following criteria are met:
```

Comment:
I feel like this might be ambiguous what we mean by forecast here. It might be possible that someone could interpret this as "forecaster" and think that we are not including any forecaster that predicts cases and deaths but also other things, or any forecaster that also predicts for horizons 5 weeks ahead, etc.

Reply:
```suggestion
A dataset is only included if all the following criteria are met:
```

?

Reply:
I'm not sure how else to phrase it. The inclusion criteria is based on (forecaster, location, horizon, target) combinations, which we defined above as a "forecast".

I don't agree with using "dataset", since there are many different things that could be considered a dataset, so I find the phrase less clear.

Reply:
That's fair, we do define forecast explicitly above. Maybe we should change the definition of "Forecaster" above to say "A model producing forecasts." That would be even clearer (and better than what I had there I think).

Reply:
Ok, I updated that and some of the surrounding terms. I think it's clearer, although the phrasing was tricky to decide.